server/test/fixtures/auth.stub.ts

```diff
@@ -9,6 +9,7 @@ const authUser = {
     isAdmin: true,
     quotaSizeInBytes: null,
     quotaUsageInBytes: 0,
+    appVersion: '1.0.0',
```

Comment:
I haven't actually checked but I'd expect we send `v1.0.0`? If so we should update the stubs to more closely resemble the reality.